README.md

```diff
@@ -68,9 +69,33 @@ You may also choose to manage the containers yourself with the `docker-compose.y
 - Default login: `admin`
 - Default password: `admin`
 
-### Commands
 
-#### Init
+### Initialization
+
+**source{d} CE** can be initialized from 2 different data sources: local Git repositories, or GitHub organizations.
+
+Please note that you have to choose one data source to initialize **source{d} CE**, but you can have more than one isolated environment, and they can have different sources. See the [Working With Multiple Data Sets](#working-with-multiple-data-sets) section below for more details.
+
+#### From GitHub Organizations
+
+When using GitHub organizations to populate the **source{d} CE** database you only need to provide a list of organization names, and a [GitHub personal access token](https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/). The token should have the following scopes enabled:
+
+- [x] `repo`  Full control of private repositories
```

Comment:
actually user can use token without any permissions than only open-source data will be downloaded. It's how I use srcd-ce. Not sure if we should mention it.

Reply:
actually, maybe we should if we use `src-d,bblfsh` as example. There is no need for any permissions because external users aren't part of src-d or bblfsh orgs.

Reply:
:thinking: but even if it's _technically_ possible, I'm not sure we want to document it.
I mean the hassle for the user that needs to create a token it's the same, we are not documenting a more convenient way. If we document private vs public, we may end up causing confusion,. Someone might use a token without access to private repos, and next week complain that the dashboards does not reflect those repos...

Reply:
Thinking a bit more about this:
This is intended to be friendly to anyone, not just developers. If I log into GitHub with my account, and I see private and public repos in some organizations, and public repos only on others. When I tell `sourced` to download repos from X,Y organization using authentication created from my account, my reference is what I see in GitHub. The less surprising result is to have the same repos, public and private.

If we want to provide a way to skip private repos, I think it should not be done tweaking the permissions of the GH token. It could be a setting from the CLI, or a filter in the dashboards.
But if we give here 2 options, and a user chooses permissions for public repos because they will maybe test over `src-d`, when they use the same token 2 weeks from now, they will forget about this detail, and they will not have visibility over why `sourced` is not showing private repositories.

If we could choose _without authentication_ and _with authentication_, that would be great. But if we ask to go over the process to create the token, let's just go with one official way to do it, that we know will work as is expected by most people.

Reply:
I'm going to merge, please feel free to open a new issue to continue this discussion if you still think it's better to change it.
It's a minor detail easy to change later.

Reply:
Yeah I'm okay to discuss it later. I paid attention to it because I'm usually very careful with giving permissions to anything. If it's possible to avoid asking for extra permissions I always would prefer to do it.